Automattic/wp-calypso
# Email stats: Show clicks in the chart component

#### Proposed Changes

* This PR adds the clicks data to the chart

<img width="1101" alt="CleanShot 2023-01-26 at 11 53 57@2x" src="https://user-images.githubusercontent.com/528287/214818652-fe931ed7-a9cc-4c9a-97d9-daa430017886.png">


#### Testing Instructions

* Apply this PR
* Apply this diff (if it hasn't landed yet) to your sandbox, and make sure `public-api.wordpress.com` is sandboxed: D99339-code
* Visit a post which has email clicks: `http://calypso.localhost:3000/stats/email/clicks/<site>/day/<postid>?stats=newsletter%2Fstats``

The chart should be populated with click data.

#### Pre-merge Checklist





Related to #
